### ISSUE TITLE
Tor not available

### ISSUE THREAD

**Issue by rllola** [krixano/ZeroNet-Windows-Exe-Installer#6]
By default Zeronet on windows install and start with TOR. This is not the case here ?

**Comment by vRobM** [comment]
you can see the errors if you run the ZeroNet.cmd manually with the command window open

**Comment by krixano** [comment]
Yes, I know about this. I'm getting ZeroNet directly from ZeroNet repo rather than ZeroNet windows bundle and I *think* the repo doesn't have tor packaged or something, idk.
